Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/SonarScanner.MSBuild.Tasks.IntegrationTests.csproj

```diff
@@ -41,17 +41,20 @@
     <Reference Include="FluentAssertions, Version=5.9.0.0, Culture=neutral, PublicKeyToken=33f2691a05b67b6a, processorArchitecture=MSIL">
       <HintPath>..\..\packages\FluentAssertions.5.9.0\lib\net47\FluentAssertions.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Bcl.AsyncInterfaces, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
```

Comment:
There are a couple of extra package references like this one, `System.Text.Json`etc that it's not obvious why they have been added. I'm assuming they are transitive dependencies of one of the new MSBuild packages.

I updated the MSBuild as follows:
* deleted all of the MSBuild package refs from the project file and the package.config.
* Used the VS UI to add a reference to a single package: `Microsoft.MSBuild.Runtime, v16.9`.

This is what I ended up with - no manual edits.